### ISSUE TITLE
Enhance the "Help" page with search-in-docs capability and better Freshdesk integration in contact form

### ISSUE THREAD

**Issue by iamronen** [opencollective/opencollective#6442]
## User story 

> I am an overloaded platform/fiscal host support person. Many of the support requests I handle end up being referring people to documentation. In other cases the first thing I need to do is to dig up additional context from the platform in order to handle or redirect the support request to the correct people.

## Best solution for this problem
This project aims to reduce the support load and improve support response times by making two changes to the "contact us" page:

1. Integrating documentation search directly into the page #5022 . This should bypass the need to open tickets for questions that can be resolved by reading documentation.
2. Automatically adding to Freshdesk tickets contextual information that is linked to the platform (user profile, collective information, fiscal host information, etc.). This requires that we create Freshdesk issues through their API #5023. This has recently become possible because we've upgraded to a paid plan. This should:
    - Enable setting up better Freshdesk automation using the added contextual information.
    - Link the support request directly to the platform so that support people can easily access the information they need in order to resolve the support request.

P1 high frequency, high impact

**Comment by BenJam** [comment]
demo over in https://around.co/playback/76274d56-cd24-449f-92fe-e0735e1f2705?sharedKey=01640aa3-2d28-4bd1-a8f1-316b81d9d9ad (4minutes in)

I like the approach here because it stops us having to second guess what we might have to attach to reports using the Freshdesk API but it does require us building tooling for specific scenarios. 

There are a few things to scope out here based on our understanding of users' most common requests/needs. 

@jphill and @shannondwray perhaps the two of you could have a look at that this week for an hour?

**Comment by BenJam** [comment]
- [ ] amina to coral shannon and john to talk about common problems, and information that would be useful to capture/attach to freshdesk ticket through API (or email if necessary)

**Comment by aminakazi** [comment]
The files shown in the demo can be [found here](https://www.figma.com/file/KinuveYf4WUlRIXzMTnmIsmO/%5BOC-Design%5D-Lab-%F0%9F%A7%AA?node-id=19309%3A149521&t=cO29GtiE8MIXJFSJ-11).

**Comment by shannondwray** [comment]
Can I also request an investigation into how the form gets delivered to Freshdesk? Ideally, it needs to be going to an inbox that is linked to Freshdesk. Currently, the form doesn't have an autoresponse, and I want to set this up. 

I was also curious if a form submitter gets an email detailing their message? - https://github.com/opencollective/opencollective/issues/4699#issue-999720213 (view the user story here)

**Comment by BenJam** [comment]
notes from conversation:

- need to direct contacts to the right collect, host or the platform team
- need to notify the reporter that their request will be sent to collective/host/platform
- need to balance getting specific information with the complexity of the form
- probably best to start with a simpler 'what do you need help with' prompt that allows us to direct their request, and to ask for specific information in the main body of request.
- can then extend with specific requests if needed, probably starting with a collective/org picker

**Comment by znarf** [comment]
- [ ] review Search in Docs integration with Olu
- [ ] sync Benjam and François on Freshdesk integration

**Comment by BenJam** [comment]
I'm just looking at some of the more common queries that we'll want to account for in the help form. Looking at [analysis from Open Collective Inc, Open Source Collective, Open Collective Foundation and OCEU documentation](https://docs.google.com/spreadsheets/d/1tbu3oDORLJyqF670sVH5ELzVtv70lCmaxEpulICrk_M/edit#gid=1445610247) (page hits and searches) 

### Page Hits

- making payments, submitting expenses, and documentation for hosts and collectives (unhelpful FAQs) are top of the list for the platform
- how to apply, fees, and service pages all rank in top ten for hosts

### Searches

- tax is the top hit
- Paypal, stripe, wise, and payoneer all appear in top ten as a platform
invoice, cancel, expense, payment, recurring all appear in top 15
- archive, password, email all appear in top 20 (change email is in top 30)
- grant, check, cash, insurance and virtual cards appear as top searches for OCF
- github, trademark, virtual, employment all appear in top for OSC

### Support requests

We're still waiting on some analysis from support, but anecdotally a lot of requests can be solved by linking to documentation, and others are about weird payment processing quirks. We also get a lot of support requests that are directed to a collective.

**Comment by BenJam** [comment]
> Can I also request an investigation into how the form gets delivered to Freshdesk?

form is sent to support@ @shannondwray

**Comment by BenJam** [comment]
One of the things we could _also_ do is implement our search-in-docs feature alongside the topic-based selection. Using the topic to fire some keywords into the search to present potentially useful results. 

For instance: 'Virtual card transaction declined' could prompt a search in docs for virtual card limits...

**Comment by BenJam** [comment]
closing this in favour of follow up work in https://github.com/opencollective/opencollective/issues/6672
